pkgs/applications/audio/dsf2flac/default.nix

```
@@ -0,0 +1,39 @@
+{ stdenv, fetchFromGitHub, autoreconfHook, boost, flac, id3lib, pkgconfig
+, taglib, zlib }:
+
+stdenv.mkDerivation rec {
+
+  pname = "dsf2flac";
+  version = "0.1Rev54";
+
+  src = fetchFromGitHub {
+    owner = "hank";
+    repo = pname;
+    rev = "b0cf5aa6ddc60df9bbfeed25548e443c99f5cb16";
+    sha256 = "15j5f82v7lgs0fkgyyynl82cb1rsxyr9vw3bpzra63nacbi9g8lc";
+  };
+
+  buildInputs = [ boost flac id3lib taglib zlib ];
+
+  nativeBuildInputs = [ autoreconfHook pkgconfig ];
```

canonical name > alias
```suggestion
  nativeBuildInputs = [ autoreconfHook pkg-config ];
```